pages/search.js

```diff
@@ -19,23 +18,14 @@ Page.propTypes = {
 };
 
 export async function getServerSideProps({ query }) {
-  console.log(query);
-  const queryElements = [];
-  for (const [key, value] of Object.entries(query)) {
-    const element = key + "=" + value;
-    queryElements.push(element);
-  }
-  const queryString = queryElements.join("&");
-  console.log(queryString);
+  const queryString = getQueryStringFromServerQuery(query);
   const url = API_URL + "/search?" + queryString;
-  console.log(url);
-
   const response = await fetch(url);
   const data = await response.text();
   const breadcrumbs = [
     {
       title: "Search",
-      href: "query",
+      href: "/search?" + queryString,
```

Comment:
One choice to make is whether to allow string concatenation through addition or not. Our ENCODE rules don’t allow this — you have to use string templates. I didn’t realize the igvf-ui one did allow this, so I’ll change that. But you might decide to allow it for RegulomeDB.